examples/gaussian_process/plot_gpr_prior_posterior.py

```diff
@@ -196,23 +196,30 @@ def plot_gpr_samples(gpr_model, n_samples, ax):
 # %%
 # Dot-product kernel
 # ..................
+from sklearn.gaussian_process import GaussianProcessRegressor
 from sklearn.gaussian_process.kernels import ConstantKernel, DotProduct
+from sklearn.preprocessing import StandardScaler
+import matplotlib.pyplot as plt
```

Comment:
`matplotlib.pyplot` is already imported in line 33
`GaussianProcessRegressor` is already imported in line 100

```suggestion
from sklearn.gaussian_process.kernels import ConstantKernel, DotProduct
from sklearn.preprocessing import StandardScaler
```